script/popup.js

```diff
@@ -0,0 +1,240 @@
+    tech: ['html', 'css', 'javascript'],
+    imageMob: './assets/card/Snapshoot-mobile-Portfolio-1.svg',
+    imageDesk: './assets/card/Snapshoot-desktop-Portfolio-3.svg',
+    live: 'https://melkamu372.github.io/Portfolio-mobile-first/',
+    source: 'https://github.com/melkamu372/Portfolio-mobile-first',
+  },
+  {
+    id: 3,
+    title: 'Multi-Post Stories',
+    canopy: ['Uber Navigation', 'Back End Dev', '2015'],
+    dot: './assets/icons/Counter.svg',
+    class: 'work-container work-container-reverse',
+    para1:
+      'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    para: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent",
+    para2:
+      "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum han printer took a galley of type and scrambled it 1960s with the releawn printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea",
+    tech: ['html', 'css', 'javascript', 'ruby'],
+    imageMob: './assets/card/Snapshoot-mobile-Portfolio-2.svg',
+    imageDesk: './assets/card/Snapshoot-desktop-Portfolio-4.svg',
+    live: 'https://melkamu372.github.io/Portfolio-mobile-first/',
+    source: 'https://github.com/melkamu372/Portfolio-mobile-first',
+  },
+];
+
+document.querySelector('.card').innerHTML = projectDatas
+  .map(
+    (data) => `
+    <section class="${data.class}" id="portfolio">
+        <div class="project-img card-card-section-1">
+          <img class='image-mobile' src="${data.imageMob}" alt="work background">
+          <img class='image-dektop' src="${data.imageDesk}" alt="work background">
+        </div>
+
+      <div class="project-card-container card-section-1">
+          <div class="projectone-header">
+            <h2>${data.title}</h2>
+          </div>
+          <div class="project-one-header-desktop">
+            <h2>${data.title}</h2>
+          </div>
+
```

Comment:
Nice job generating your project section dynamically😉. I suggest you kindly consider the spacings around the project section both on mobile and on desktop as your project is looking a bit different from the given design currently😉. see the image below😉.

yours
<img width="1432" alt="image" src="https://github.com/melkamu372/Portfolio-mobile-first/assets/64643995/50312560-181b-43be-9672-b2ce39031f30">


design
<img width="727" alt="image" src="https://github.com/melkamu372/Portfolio-mobile-first/assets/64643995/3aa8bc4f-d2c0-4abd-a9f5-e37a47a5bb4a">